about/map-parpax.txt

```diff
@@ -242,6 +242,10 @@ sounds/parpax/elevator.flac
 ? / Delta 5
     - ?
 
+09.07.2020 / 2.6.0 (Delta 6)
```

Comment:
Note: maybe it would be better to switch to ISO's YYYY-MM-DD later but there is no need to pollute this PR with that.

Reply:
Sure, just being consistent here.